netsim/providers/__init__.py

```diff
@@ -163,7 +163,9 @@ def create_extra_files(
       file_name = file.replace(out_folder+"/","")
       template_name = self.find_extra_template(node,file_name,topology)
       if template_name:
-        node_data = node + { 'hostvars': topology.nodes, 'hosts': get_host_addresses(topology) }
+        node_data = node + { 'hostvars': topology.nodes, 
+                             'hosts': get_host_addresses(topology),
+                             'mgmtpool': topology.addressing.mgmt }  # Needed for subnet prefix
```

Comment:
Is there a reason you're not copying 'topology.addressing' into 'addressing'? That would give us more information in case we ever need it.

Reply:
done